src/components/Match/Laning/index.jsx

```diff
@@ -32,8 +33,29 @@ class Laning extends React.Component {
       match, strings, sponsorURL, sponsorIcon,
     } = this.props;
     const { laningColumns } = mcs(strings);
+    const annotatedPlayers = match.players.map((p) => {
```

Comment:
We have a central "post-fetch" transform function where we do these computations on the match object.

See https://github.com/odota/web/blob/master/src/actions/transformMatch.js#L146